server/src-lib/Hasura/Server/Init.hs

```diff
@@ -1098,10 +1098,12 @@ downgradeShortcuts =
   , ("v1.0.0", "28")
   , ("v1.1.0-beta.1", "29")
   , ("v1.1.0-beta.2", "30")
+  , ("v1.1.0-beta.3", "31")
```

Comment:
Looks good, thanks!

Reply:
FYI I generated these with something like `git show v1.1.0:./src-lib/Hasura/Server/Migrate/Version.hs | grep 'latestCatalogVersion ='` which might be useful in future.